### PR TITLE
Release note: changes to Discovery API endpoints

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Release note that describes the deprecation of certain endpoints and query patterns in the dbt Cloud Discovery API.

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Needs review from PM

Adding new pages (delete if not applicable):
- [x] Add page to `website/sidebars.js`
- [x] Provide a unique filename for the new page

### CODE REVIEW

File: website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+sidebar_label: "Deprecation: Certain Dicovery API endpoints and query patterns"
+tags: [Aug-2023, API]
+date: 2023-08-31
+---
+
+dbt Labs has deprecated and will be deprecating certain query patterns and replacing them with new conventions to enhance the performance of the dbt Cloud [Discovery API](/docs/dbt-cloud-apis/discovery-api). 
+
+All these changes will be in effect on _September 7, 2023_. 
+
+We understand that these changes might require adjustments to your existing integration with the Discovery API. Please [contact us](mailto:support@getdbt.com) with any questions. We're here to help you during this transition period.
```
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md:14]: @cafzal : is linking to support@getdbt.com the correct one to use?

Comment (cafzal) [reply website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md:14]: yes

File: website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
+---
+title: "Query patterns and endpoints in the dbt Cloud Discovery API"
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md]: Should we use "Deprecation:" in the title too for clarity?

File: website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+description: "August 2023: Learn about the upcoming deprecation of certain endpoints and query patterns in the Discovery API."
+sidebar_position: 6
+sidebar_label: "Deprecation: Certain Dicovery API endpoints and query patterns"
+tags: [Aug-2023, API]
+date: 2023-08-31
+---
+
+dbt Labs has deprecated and will be deprecating certain query patterns and replacing them with new conventions to enhance the performance of the dbt Cloud [Discovery API](/docs/dbt-cloud-apis/discovery-api).
```
Comment (runleonarun) [inline website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md:10]: Should we mention why we're making this change?

**Review by runleonarun — APPROVED** [review]
@nghi-ly two non-blocking comments.

File: website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md (hunk 2 above)
Comment (nghi-ly) [inline website/docs/docs/dbt-versions/release-notes/05-Aug-2023/deprecation-endpoints-discovery.md]:
```suggestion
title: "Deprecation: Query patterns and endpoints in the dbt Cloud Discovery API"
```

**Pull request merged**